tutorials/cloud-iot-logging/index.md

```diff
@@ -0,0 +1,238 @@
+
+![console image](./images/c2.png)
+
+Which then looks like this:
+
+![console image](./images/c3.png)
+
+### Combine system and Applications logs
+
+Now we will exercise a part of our sample device code that responds to config changes. Use the following gcloud command to update the devices config telling it to "bounce to a level of 2":
+
+```sh
+gcloud iot devices configs update --device log-tester --registry $REGISTRY_ID --region $CLOUD_REGION --config-data '{"bounce": 2}'
+```
+
+Now in just a few moments, you will see two new entries in the logging console. One is from IoT Core system noting that a devices config was updated (the ModifyCloudToDeviceConfig call).
+
+This is then followed by a device application log reporting the imaginary "spring back" value. This shows how we can view both system logs from IoT Core and device application logs in one place.
+
+You can use the refresh button in the Cloud Console, or use the play button to stream logs.
+
+![console image](./images/c4.png)
+
+### Use severity to convey log level
+
+Lets send a couple more config updates (you should send one, wait a second or two, then send the next):
+
+```sh
+gcloud iot devices configs update --device log-tester --registry $REGISTRY_ID --region $CLOUD_REGION --config-data '{"bounce": 7}'
+
+# pause
+
+gcloud iot devices configs update --device log-tester --registry $REGISTRY_ID --region $CLOUD_REGION --config-data '{"bounce": 17}'
+
+```
+
+As these are received and responded to by the sample device, you can see the use of a severity level to indicate the importance of the log.
+
+![console image](./images/c5.png)
+
+### Filter logs to a specific device
+
+We currently have been looking at a log that contains entries for all devices.  But given these are structured log entries, we can use the numeric id of the resource to limit our view to only one device in the more realistic scenario when multiple devices are writing log events.
+
+We open up the log entry, find the resource, labels, and choose `device_num_id`, click on it and choose `Show matching entries`:
+
+![console image](./images/c6.png)
+
+This creates a quick filter of the log:
+
+![console image](./images/c7.png)
+
+You can get a sense the filter syntax. You can even do things like a substring match on a field in the json payload of the log message, try adding `jsonPayload.msg:"Spring"`. See the [docs](https://cloud.google.com/logging/docs/view/advanced-filters) for more on using advanced filters.
+
+
+## Cleaning up
+
+Kill the sample device:
+
+```sh
+killall node
+```
+Because the test device uses a non-secret key, you should delete it:
+
+```sh
+gcloud iot devices delete log-tester --registry $REGISTRY_ID --region $CLOUD_REGION
+```
+
+All of the resource in this tutorial cost nothing at rest, or scale to zero.  You can delete Cloud Functions, Device Registry, and PubSub topics from the console.
```

Comment:
I'm not sure whether these instructions are likely to be clear enough for your intended audience. Can you provide links to the appropriate console pages? Or otherwise clarify the process? Also, it's a little vague whether/why you would want to delete these resources, if they persist without cost.